lib/MusicBrainz/Server/Report/LimitedEditors.pm

```diff
@@ -0,0 +1,33 @@
+package MusicBrainz::Server::Report::LimitedEditors;
+use Moose;
+
+use MusicBrainz::Server::Constants qw( $EDITOR_MODBOT );
+
+with 'MusicBrainz::Server::Report::EventReport';
+
+sub query { "
+SELECT id, name, email, website, bio, member_since, email_confirm_date,
+       last_login_date, last_updated, deleted,
+       row_number() OVER (ORDER BY id DESC)
+  FROM editor eor
+ WHERE id != $EDITOR_MODBOT
+   AND NOT deleted
+   AND   ( email_confirm_date IS NULL
+        OR member_since < NOW() - INTERVAL '2 weeks'
+        OR (SELECT COUNT(*) FROM edit WHERE eor.id = edit.editor AND edit.status = 2 AND edit.autoedit = 0) < 10
+         )";
```

Comment:
Is it currently unordered, or ordered by name elsewhere? Should we order by `member_since DESC` instead? It seems like it'd be useful for admins to see the most recent accounts at the top of the list.

Reply:
I guess @Freso or @reosarevok will be looking at this report the most, so however they'd want it ordered.

Reply:
Yes. Actually, rows are ordered by `id` at [line 11](https://github.com/metabrainz/musicbrainz-server/pull/556/files/cc1a4fdc419081b809f511d471435208a00b0d5e#diff-d1da13642c99eb0de1f985a77a2c6467R11). Greatest ID being the most recent, it is the same as ordering by `member_since`.